### PR TITLE
Create Brianna's logical turtle post

### PR DESCRIPTION
**Pull request by brianna-stedman**
let me know if I embedded correctly!

### CODE REVIEW

**Comment by brianna-stedman** [comment]
whoops I gotta add a reflection

File: _posts/brianna-stedman/09-09-2020-brianna-stedman-logical-turtle.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+layout: post
+author: brianna-stedman
+title: "Brianna's logical turtle"
+---
+
+What is Tina's favorite color? Give it your best guess! If you're correct, she'll do a little happy dance. 
+
+<iframe src="https://trinket.io/embed/python/6ea79a3db4" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
```
Comment (KJEZ35) [inline _posts/brianna-stedman/09-09-2020-brianna-stedman-logical-turtle.md:9]: I did the same thing, I hope this is what he wanted!

**Review by KJEZ35 — CHANGES_REQUESTED** [review]
test

File: _posts/brianna-stedman/09-09-2020-brianna-stedman-logical-turtle.md (hunk 1 above)
Comment (brianna-stedman) [reply _posts/brianna-stedman/09-09-2020-brianna-stedman-logical-turtle.md:9]: okay sweet, I think it's good to go if you wanna approve the change and merge it!

**Comment by brianna-stedman** [comment]
I reviewed change request, should be good to merge now, thanks @KJEZ35 !

**Review by KJEZ35 — APPROVED** [review]
Everything looks great Brianna!

**Comment by brianna-stedman** [comment]
@eah13 I'm looking back over this and it appears as though Kevin merged it correctly, but I'm not seeing this post under my name on the People page on our course website. I've looked over my post/file name and it all *looks* right to me, but I'm also not sure what "wrong" looks like to spot it. Would you be able to take a look at this?

**Comment by brianna-stedman** [comment]
oh, and likewise - I thought I had successfully merged @KJEZ35's post in return, but it also is not showing up under his name on the People page. I think we might not be on the same page @eah13 do know know where we went wrong here?

**Pull request merged**